be/src/agent/agent_server.cpp

```diff
@@ -411,6 +434,7 @@ ThreadPool* AgentServer::Impl::get_thread_pool(int type) const {
     case TTaskType::DROP:
     case TTaskType::PUSH:
     case TTaskType::CLONE:
+        return _thread_pool_clone.get();
     case TTaskType::STORAGE_MEDIUM_MIGRATE:
```

Comment:
CREATE/DROP/PUSH/CLONE all use this pool?

Reply:
That's a mistake, i'll fix it.